deploy/jenkins/jenkins-server-setup.sh

```diff
@@ -18,7 +18,10 @@ apt-get update
 apt-get install -y jenkins=2.263.4
 
 echo -e "\n\e[0;32m${bold}Installating PIP${normal}"
-apt-get install -y python-pip
+udo apt update 
+sudo apt install python2
+curl https://bootstrap.pypa.io/2.7/get-pip.py --output get-pip.py
+sudo python2 get-pip.py
```

Comment:
why can't we just use `sudo apt install python3-pip` ?

Reply:
@rjshrjndrn We need PIP latest version. If you do sudo apt install python3-pip then it will install pip version 8X, for which the ansible is not getting installed when we run the jenkins-server-installation script. So we are adding these steps to install the pip latest version 20X.